test/e2e-common.sh

```diff
@@ -186,41 +186,31 @@ function install_knative_serving() {
 }
 
 function install_istio() {
-  # If no gateway was set on command line, assume Istio
   if [[ -z "${ISTIO_VERSION}" ]]; then
-    echo ">> No gateway set up on command line, using Istio"
-    readonly ISTIO_VERSION="1.4-latest"
+    readonly ISTIO_VERSION="stable"
   fi
 
-  local istio_base="./third_party/istio-${ISTIO_VERSION}"
-  INSTALL_ISTIO_CRD_YAML="${istio_base}/istio-crds.yaml"
+  local NET_ISTIO_DIR=$(mktemp -d)
+  git clone --quiet --depth 1 --branch master https://github.com/knative-sandbox/net-istio.git ${NET_ISTIO_DIR}
```

Comment:
While unrelated to our P0, the hardcoded "master" here is going to be a problem.